emma-flink/src/main/scala/org/emmalanguage/api/FlinkDataSet.scala

```diff
@@ -78,6 +77,13 @@ class FlinkDataSet[A: Meta] private[api](@transient private val rep: DataSet[A])
   // Sinks
   // -----------------------------------------------------
 
+  override def writeCSV(path: String, format: CSV)(implicit converter: CSVConverter[A]): Unit =
```

Comment:
This ignores some fields of the `format`. Shouldn't this method fail if the value of these fields is not the default value?